src/modules/iotjs_module_stm32f7nucleo.c

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2019-present Samsung Electronics Co., Ltd. and other contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "iotjs_def.h"
+#include "iotjs_module_stm32f7nucleo.h"
+
+
+jerry_value_t InitStm32f7nucleo() {
```

Comment:
Why is this function "empty"? I don't see any added value in this.

Reply:
Since its something generic I decided to introduce it in intial patch, it's here for upcoming initialisations (I am trying to do PWM ATM).

Reply:
Please add some comment at least, good for me after that.